### ISSUE TITLE
Add seo_description frontmatter to all content

### ISSUE THREAD

**Issue by mmontesanonyc** [nycehs/ehs-data-portal-frontend-temp#64]
Note the frontmatter for neighborhood_reports content: it includes a `seo_description` field. 

This field is read by the `seo.html` partial, which sets values/content for Open Graph content (this is like preview content when a link is shared in social media). There are conditionals in here that we can modify about what main content or backup content is used for this Open Graph content. But, a big improvement we can make is to add an seo_description to every piece of content - data stories, key topics, data features, and data-explorer subtopics (NRs are already done). 

And, we can look deeper into SEO-specific frontmatter fields that were built for the neighborhood reports and see about replicating those site-wide, too.

**Comment by mmontesanonyc** [comment]
Additional ways to improve SEO, per our contractor's report:
- Shorten page titles to 60 characters
- Give pages unique Meta descriptions
- Give all pages keywords
- Key Topics template: ensure that this has an H1/H2/H3 properly nested.
- Re-write image filenames to be more descriptive
- Review broken links report
- Change categories to have a hyphen instead of an underscore (e.g, data_stories, data_explorer) - and, accordingly, revise ingestions (related.html partial).

We have a downloaded packet of SEO review from contractor to review before launching this work.

**Comment by mmontesanonyc** [comment]
Bumping this - @cgettings let's chat about this in our meeting today - there are some things on here that it would be good to do before we start working on the Data Viz module with the contractor.
